template/core/commons/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
+    id("template.kotlin.feature")
+}
+
+dependencies {
+
```

Comment:
libs.javax.inject and base dagger dependencies could be here, and passed along as `api` to other modules? Or is that a bad idea?